package.json

```diff
@@ -30,7 +30,7 @@
     "test": "jest",
     "test:ci": "jest --outputFile test-results.json --json --runInBand",
     "testing": "jest --watch",
-    "relay": "relay-compiler --src ./src --schema data/schema.graphql --language typescript --artifactDirectory ./src/__generated__ --persist --persistOutput ./src/__generated__/complete.queryMap.json",
+    "relay": "relay-compiler --src ./src --schema data/schema.graphql --language typescript --artifactDirectory ./src/__generated__ --exclude '**/node_modules/**,**/__mocks__/**,**/__generated__/**'",
```

Comment:
`relay-mock-network-layer`, used in [`createMockNetworkLayer`](https://github.com/javamonn/emission/blob/806b98a4ed81ce54f354a2459ba79a95171d7eaa/src/lib/tests/createMockNetworkLayer/index.ts#L16), depends on [query text being present](https://github.com/1stdibs/relay-mock-network-layer/blob/master/index.js#L21) in the operation in order to client side resolve. Our current approach of used persisted queries however sets `text` to `null` in generated `.graphql` artifacts so that queries are not sent over the network. We obviously don't want to disable persisted queries just to get these tests to work.

Reply:
Some ideas @orta had to workaround this: 
- Update compiler to include query text as some other field in the generated artifact, presumably wouldn't be sent over the wire at runtime as the runtime wouldn't know about the generated field.
- Update `relay-mock-network-layer` to resolve operation text differently